sdk/keyvault/Microsoft.Azure.KeyVault/tests/Microsoft.Azure.KeyVault.Tests.csproj

```diff
@@ -27,6 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
```

Comment:
What is this needed?

Reply:
Yes, for the callback that `KeyVaultClient` needs. Any concern?

Reply:
This is track 1 and a test project so I'm not concerned with this. We do need to be a little careful about introducing dependencies though.